src/redisraft.c

```diff
@@ -1297,6 +1297,111 @@ static int cmdRaftRandom(RedisModuleCtx *ctx,
     return REDISMODULE_OK;
 }
 
+static int cmdRaftScan(RedisModuleCtx *ctx,
+                         RedisModuleString **argv, int argc)
+{
+    if (argc < 3) {
+        RedisModule_WrongArity(ctx);
+        return REDISMODULE_OK;
+    }
+
+    char cursor[15] = {'0', 0};
```

Comment:
```suggestion
    char cursor[15] = {0};
```

Reply:
use the '\0' instead of 0 for the string

Reply:
not needed anymore as no more default cursor, have to specify 0, so can just zero whole thing out.